app/models/manageiq/providers/network_manager.rb

```diff
@@ -44,5 +46,13 @@ class << model_name
              :hosts,
              :to        => :parent_manager,
              :allow_nil => true
+
+    def self.supported_types_and_descriptions_hash
+      supported_subclasses.select { |x| x.supports_create? }.each_with_object({}) do |klass, hash|
```

Comment:
rubocop marks this

you can write 
supported_subclasses.select { |x| x.supports_create? }
as
supported_subclasses.select(&:supports_create?)